index.html

```diff
@@ -1,70 +1,70 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Awesome Books</title>
-  </head>
-  <body>
-    <header>
-      <nav class="navbar">
-        <div class="header-top">
-          <h1 class="title1">Book Store</h1>
-        </div>
-        <ul>
-          <li><a class="links" id="listbtn">List</a></li>
-          <li><a class="links" id="add-newbtn">Add New</a></li>
-          <li><a class="links" id="contactbtn">Contact</a></li>
-        </ul>
-      </nav>
-      <div class="timer">00:00:00</div>
-    </header>
-    
-    <section class="books" id="list">
-      
-      <div class="books-wrapper" id="books-wrapper">
-        <h2 class="list-header">List</h2>
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesome books</title>
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <<nav class="nav-bar">
```

Comment:
- [ ] Kindly remove the extra `<` at the start of this line as it is a typo and can be seen on the page clearly.

Reply:
fix it thank you 👏
 You can find this project here: [https://github.com/kayjeee/bookstore].  Please leave a star it helps. Thanks for checking out my code!